quinn-proto/src/frame.rs

```diff
@@ -738,6 +738,9 @@ impl NewToken {
 pub(crate) struct MaxPathId(pub(crate) PathId);
 
 impl MaxPathId {
+    pub(crate) const SIZE_BOUND: usize =
+        VarInt(FrameType::MAX_PATH_ID.0).size() + VarInt(u32::MAX as u64).size();
```

Comment:
is the second part not the same as 
```suggestion
        VarInt(FrameType::MAX_PATH_ID.0).size() + VarInt::MAX_SIZE;
```

Reply:
Practically yes, because 2**32 is represented in 8 bytes in a QUIC VarInt.  But I thought I'd go with the logical representation: the maximum value possible in that field is a PathId, so a u32.  So that's what I'm calling, in the believe this represents the intention a little better.

(fwiw the existing code often hardcodes a lot of this stuff.  I started doing this const stuff because the frame types are likely to change from needing 4 bytes to 1 byte once the spec is out of draft.  But I like it in general instead of writing the magic numbers.  Of course there is also the consideration on whether to use actual values, many PathIds will probably fit in one byte in reality... But let's leave that for later.)